malaria-ios/Information/EndpointType.swift

```diff
@@ -21,10 +21,14 @@ import Foundation
 /// - `Objectives`
 /// - `Goals`
 public enum EndpointType : String{
-    case BaseUrl = "http://pc-web-dev.systers.org/"
+    #if DEBUG
+    case BaseUrl = "https://systerspcweb.herokuapp.com/"
+    #else
```

Comment:
Thank you for your response. Sorry for my poor English.
It is not urgent, of course.
I look forward to your feedback.